cpp/ql/lib/semmle/code/cpp/models/interfaces/Throwing.qll

```diff
@@ -11,12 +11,21 @@ import semmle.code.cpp.models.Models
 import semmle.code.cpp.models.interfaces.FunctionInputsAndOutputs
 
 /**
- * A class that models the exceptional behavior of a function.
+ * A function that is known to raise an exception.
  */
 abstract class ThrowingFunction extends Function {
+  ThrowingFunction() { any() }
+
   /**
    * Holds if this function may throw an exception during evaluation.
    * If `unconditional` is `true` the function always throws an exception.
    */
   abstract predicate mayThrowException(boolean unconditional);
 }
+
+/**
+ * A function that is known to raise an exception unconditionally.
+ * The only cases known where this happens is for SEH
+ * (structured exception handling) exceptions.
+ */
```

Comment:
Since "SEH" is in the name of the class, I would change this to just:
```suggestion
/**
 * A function that is known to unconditionally raise an structured exception handling (SEH) exception.
 */
```